spring/boot2-actuator-autoconfigure/src/test/java/com/linecorp/armeria/spring/actuate/ArmeriaSpringActuatorAutoConfigurationInternalServiceTest.java

```diff
@@ -195,21 +200,31 @@ static class BasePathSamePortTest {
         @Inject
         private InternalServices internalServices;
 
+        @DynamicPropertySource
+        static void registerPortProperties(DynamicPropertyRegistry registry) {
+            final int port = SocketUtils.findAvailableTcpPort();
```

Comment:
This will be deprecated at 6.0. But we have another test using this too. So we can replace all of them together when upgrading to Spring 6.0.

Reply:
`SocketUtils` will be replaced with `PortUtil` in #4574